lib/bitcoin/BitcoinClient.ts

```diff
@@ -565,13 +565,13 @@ export default class BitcoinClient {
 
     freezeTransaction.fee(transactionFee);
 
-    return [freezeTransaction, freezeScript.toHex()];
+    return [freezeTransaction, freezeScript];
   }
 
   private async createSpendToFreezeTransaction (
     previousFreezeTransaction: BitcoreTransactionWrapper,
     previousFreezeDurationInBlocks: number,
-    newFreezeDurationInBlocks: number): Promise<[Transaction, string]> {
+    newFreezeDurationInBlocks: number): Promise<[Transaction, Script]> {

```

Comment:
For my understanding: what's the benefit for returning `Script` instead of hex string here?

Reply:
Need to pass `Script` to the `IBitpayWallet` APIs. There is an opportunity to update the `IBitcoinWallet` to remove the requirement of bitcore-lib's objects but that's a separate issue that I opened sometime ago.